player.go

```diff
@@ -62,6 +63,21 @@ type CurrentlyPlaying struct {
 	Item *FullTrack `json:"Item"`
 }
 
+type RecentlyPlayedItem struct {
+	// Track is the track information
+	Track SimpleTrack `json:"track"`
+
+	// PlayedAt is the time that this song was played
+	PlayedAt time.Time `json:"played_at"`
```

Comment:
Can we update the tests to make sure that `played_at` is unmarshaled correctly?  Other than that looks good to me.